idaes/generic_models/unit_models/column_models/solvent_condenser.py

```diff
@@ -27,11 +27,10 @@
 
 # Import Pyomo libraries
 from pyomo.environ import (
+    check_optimal_termination,
```

Comment:
Is this import necessary, given the only call to it is commented out?

Reply:
Strictly no, but that test really should be fixed and uncommented, so I left the import in place.